crates/cheatcodes/src/env.rs

```diff
@@ -263,6 +294,10 @@ fn get_env(key: &str) -> Result<String> {
     }
 }
 
+fn is_forge_context(context: &ForgeContext) -> bool {
+    discriminant(context) == discriminant(FORGE_CONTEXT.get().unwrap())
```

Comment:
derive `PartialEq, Eq` and remove discriminant calls

Reply:
updated in https://github.com/foundry-rs/foundry/pull/7377/commits/d3d661cca7cc34075d31a74023b8e34c248c0c72